Formula/llvm.rb

```diff
@@ -147,11 +188,11 @@ def install
       (buildpath/"tools/clang/tools/extra").install resource("clang-extra-tools")
     end
 
-    if build.with? "libcxx"
-      (buildpath/"projects/libcxx").install resource("libcxx")
-    end
-
+    (buildpath/"projects/libcxx").install resource("libcxx") if build_libcxx
     (buildpath/"tools/lld").install resource("lld") if build.with? "lld"
+    (buildpath/"projects/libcxxabi").install resource("libcxxabi") if build.with? "libcxxabi"
+    (buildpath/"projects/libunwind").install resource("libunwind") if build.with? "libunwind"
+    (buildpath/"projects/openmp").install resource("openmp") if build.with? "openmp"
```

Comment:
These look like they should be using iteration (`each … do`, etc.) since they're highly repetitive and all the names match up nicely.